containers/elm-0.19.0/.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,18 @@
+{
+	"name": "Elm (0.19.0)",
```

Comment:
Would you expect to create separate container definitions for different versions of Elm?  If not, we can probably just call the definition `Elm`.  Same goes for the folder name.

Reply:
Since there were breaking changes from Elm 0.18 to 0.19, I thought it would be good to indicate it. But with the Elm version as an ARG that's easily configurable, it will make sense to refactor it into Elm only!